tests/unit/test_data_loader.py

```
@@ -15,47 +17,52 @@ class TestDataLoader:
 
     @patch("builtins.open", new_callable=mock_open, read_data='{"test": "data"}')
     def test_load_json_data_success(self, mock_file):
         """Test successful JSON data loading."""
         result = load_json_data("test.json")
-        
+
         assert result == {"test": "data"}
         mock_file.assert_called_once()
 
     @patch("builtins.open", side_effect=FileNotFoundError)
     def test_load_json_data_file_not_found(self, mock_file):
```

Ruff (0.13.1)

Unused method argument: `mock_file`

(ARG002)